File: answer.py
def solution(number):
    totalSum=0
    newArr=[]
    for y in range(number-1):
        newArr.append(y+1) if (y+1)%5==0 or (y+1)%3==0 else newArr
    for x in newArr:
        totalSum+=x 
    return totalSum
  
def solution(number):
    return sum(x for x in range(number) if x % 3 == 0 or x % 5 == 0)

def solution(number):
    sum = 0
    for i in range(number):
        if (i % 3) == 0 or (i % 5) == 0:
            sum += i
    return sum
  

File: answer.js
function solution(number){
  let newArr= []
  for(let i=1;i<number;i++){
    i%5==0 || i%3==0 ? newArr.push(i) : newArr
  }
  let result = newArr.length!==0 ? newArr.reduce((a,b)=>a+b) : 0
  return result
}

function solution(number){
  var sum = 0;
  
  for(var i = 1;i< number; i++){
    if(i % 3 == 0 || i % 5 == 0){
      sum += i
    }
  }
  return sum;
}

solution= n=> n<=0?0:Array.from({length: n}, (_,i) => (i%3==0||i%5==0)?i:0).reduce((x,y)=>x+y)
